src/error_chain.rs

```diff
@@ -54,6 +58,10 @@ macro_rules! error_chain_processed {
         }
 
     ) => {
+        use ::std::fmt::Debug;
+        use ::std::error::Error as StdError;
+        create_super_trait!(Trait: Debug, StdError, Send $(, $bound)*);
```

Comment:
It will not work if you define multiple errors in the same module.
Also, can you remove `use` and use absolute path instead?